config/forms/2021_2022.json

```diff
@@ -3222,37 +3222,37 @@
                   "hint_text": "",
                   "type": "radio",
                   "answer_options": {
-                    "0": {
+                    "1": {
```

Comment:
How come this is changing? These should match the data dictionary right?

Reply:
Yes, I think that these were pre this PR 0,1,2 .... 10 and now they are matching the dictionary? Are we looking at different dictionary files?